MdePkg/Include/IndustryStandard/PciExpress60.h

```diff
@@ -116,6 +132,53 @@ typedef union {
   UINT32    Uint32;
 } PCI_REG_PCIE_DEVICE_STATUS3;
 
+typedef union {
+  struct {
+    UINT32    InterruptSupport          : 1;      // bit 0
+    UINT32    DoeInterruptMessageNumber : 11;     // bit 1:11
+    UINT32    Reserved                  : 20;     // Reserved bit 12:31
+  } Bits;
+  UINT32    Uint32;
+} PCI_EXPRESS_REG_DOE_CAPABILITIES;
+
+typedef union {
+  struct {
+    UINT32    DoeAbort           : 1;             // bit 0
+    UINT32    DoeInterruptEnable : 1;             // bit 1
+    UINT32    Reserved           : 29;            // Reserved bit 2:30
+    UINT32    DoeGo              : 1;             // bit 31
+  } Bits;
+  UINT32    Uint32;
+} PCI_EXPRESS_REG_DOE_CONTROL;
+
+typedef union {
+  struct {
+    UINT32    DoeBusy            : 1;             // bit 0
+    UINT32    DoeInterruptStatus : 1;             // bit 1
+    UINT32    DoeError           : 1;             // bit 2
+    UINT32    Reserved           : 28;            // Reserved bit 3:30
+    UINT32    DataObjectReady    : 1;             // bit 31
+  } Bits;
+  UINT32    Uint32;
+} PCI_EXPRESS_REG_DOE_STATUS;
+
+typedef struct {
+  PCI_EXPRESS_EXTENDED_CAPABILITIES_HEADER    Header;
+  PCI_EXPRESS_REG_DOE_CAPABILITIES            Capability;
+  PCI_EXPRESS_REG_DOE_CONTROL                 Control;
+  PCI_EXPRESS_REG_DOE_STATUS                  Status;
+  UINT32                                      DoeWriteDataMailbox;
+  UINT32                                      DoeReadDataMailbox;
+} PCI_EXPRESS_EXTENDED_CAPABILITIES_DOE;
+
+typedef struct {
+  UINT32    Header1;
+  UINT32    Header2;
+  UINT32    Dw0;
+} PCI_EXPRESS_DOE_DISCOVERY_PACKET;
+

```

Comment:
Is there and additional details on these 3 UINT32 values from the PCI Spec that can be added to this .h file?